packages/next-app/components/Cards/ContributionList.tsx

```diff
@@ -1,31 +1,45 @@
-import React from "react";
-import { Box, Tag, Text } from "degen";
+import { Box, Text } from "degen";
 import { Contribution } from "../../graph/loudverse-graph-types";
 import { ethers } from "ethers";
 import { toTrimmedAddress } from "../../utils";
-import Link from "next/link";
+import { useContext } from "react";
+import { CallContext } from "../FullPageProject";
 
 const ContributionList = ({ contributionList }: { contributionList: Contribution[] }) => {
+  const { cffContext } = useContext(CallContext);
+
   return (
-    <Box padding="4" backgroundColor="foregroundSecondary" borderRadius="medium">
+    <Box padding="4" height="max" backgroundColor="foregroundSecondary" borderRadius="medium">
       <Box marginBottom="4">
         <Text size="extraLarge" weight="semiBold">
           Contribution History
         </Text>
       </Box>
-      <Box overflow="scroll" height="24">
+      <Box overflow="scroll" height="44">
         {contributionList?.length > 0 ? (
           contributionList.map((contribution, i) => {
-            const date = new Date(contribution?.timestamp * 1000).toDateString().toString();
+            let date;
+            if (contribution?.timestamp !== (null || undefined)) {
+              date = new Date(contribution.timestamp * 1000).toDateString().toString();
+            } else {
+              date = new Date().toDateString().toString();
```

Comment:
Nit: the `toString()` isn't needed here.

Reply:
/facepalm. thank you!